Project.toml

```diff
@@ -57,7 +57,7 @@ NCDatasets = "0.13.1, 0.14"
 SciMLBase = "2"
 StaticArrays = "1"
 StatsBase = "0.34"
-SurfaceFluxes = "0.11"
+SurfaceFluxes = "0.11, 0.12"
```

Comment:
Do we know if have to apply any changes?

Reply:
It was hard to tell from the release notes, but I just found this comment by @kmdeck:

> @trontrytel In simulations where land is forced by a prescribed atmosphere, land needs to compute its own fluxes. In principle that means we just call SF functions, but instead right now we do that and then adjust the SF fluxes as follows: https://github.com/CliMA/ClimaLand.jl/blob/a746febe976c4dcab555c2afb8dbbe0483e169b4/src/shared_utilities/drivers.jl#L367-L372
> 
> This is actually more similar to the SF computation in this PR, so that is good. We had switched to this form because it was easier to debug/understand than the previous SF computation (e.g. SHF doesnt depend on E). But note that we also need to account for surface resistance in the fluxes, and SF does not handle that right now, so we do it "offline".
> 
> To couple the full land model to atmos, we need to adjust SF to account for resistance. We should also adjust our standalone land model to use the output of SF directly. This will make coupled runs easier to debug.

Reply:
This should not be breaking change for us in terms of interface, and it wont change ClimaLand offline simulations since we compute surface fluxes on our own (can explain more), but it may change coupled behavior